src/docs/guides/decoupled-cms/README.md

```diff
@@ -59,7 +59,7 @@ The newly created website should be able to run, and look like this:
 
 ```xml
 <PropertyGroup>
-  <TargetFramework>net6.0</TargetFramework>
+  <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
This doesn't have to be, updating to net7.0 is fine, there is no reason to have two, even more in a sample.

Reply:
rolled it back